ir-engine/ir-engine
# Ir 4123 compression with org name

## Summary

Fixes a bug that broke model compression when we nested our projects within org folders.

closes https://tsu.atlassian.net/browse/IR-4123

## QA Steps
Compress a model, drag its integrated GLTF into a scene and confirm that the resulting new object in the hierarchy contains a "LOD-variant" child that has "Variant" components pointing to the different LOD levels you generated.
